tests/testKalturaFFMpegCmdGenerator.js

```diff
@@ -0,0 +1,55 @@
+
+const chai = require('chai');
+const expect = chai.expect;
+const KalturaFFMpegCmdGenerator = require('../lib/utils/KalturaFFMpegCmdGenerator');
+const kalturaTypes = require('../lib/client/KalturaTypes');
+const ApiClientConnector = require('../lib/infra/ApiServerClientConnector');
+const KalturaMediaInfo = require('../lib/utils/KalturaMediaInfo');
+const KalturaMediaInfoResponse = require('../lib/utils/KalturaMediaInfoResponse');
+
+describe('test KalturaFFMpegCmdGenerator', function() {
+
+    let serviceUrl = KalturaConfig.config.testClient.serviceUrl;
+    let secret = KalturaConfig.config.testClient.secret;
+    let partnerId = KalturaConfig.config.testClient.partnerId;
+    let testsDirName = __dirname;
+    let flavorId = '0_t50woq53';
+    let response = null;
+    let info = new KalturaMediaInfo('ffprobe');
+    let connector = new ApiClientConnector(partnerId,secret,kalturaTypes.KalturaSessionType.ADMIN,serviceUrl);
+
+    it('test - get mediaInfo for ad', function() {

```

Comment:
this also seems like a double check - we do not test here the getMediaInfo but the CMD generator - either simulate the other one or use without testing as a pre-requisite